query/security/wcpm/src/main/java/org/codice/ddf/admin/security/wcpm/actions/discover/GetAuthTypes.java

```diff
@@ -37,11 +37,12 @@ public GetAuthTypes(ConfiguratorFactory configuratorFactory) {
 
     @Override
     public ListField<AuthType> performAction() {
-        // TODO: tbatie - 4/14/17 - Check the backend to see if these services are running
-        return new ListFieldImpl<>(AuthType.class).addAll(Arrays.asList(AuthType.BASIC_AUTH,
+        ListField<AuthType> authTypes = new ListFieldImpl<>(AuthType.class).addAll(Arrays.asList(AuthType.BASIC_AUTH,
```

Comment:
I don't know if this is the right time to address this but this needs to be dynamic. Need to figure out how to provide descriptions for these fields dynamically. Whatever the solution is for this, apply it to realms too.